tests/test_builds.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import Mock
+
+import pytest
+
+from pybuildkite.builds import Builds
+
+
+class TestBuilds:
+    """
+    Test functionality of the Builds class
+    """
+
+    @pytest.fixture
+    def fake_client(self):
+        """
+        Build a fake API client
+        """
+        return Mock(get=Mock())
+
+    def test_meta_data_url(self, fake_client):
```

Comment:
We'll probably want to add a test where `meta_data` isn't added to verify the bug I listed above no longer exists

Reply:
Yup, added! And in fact it would fail :)